src/AI/CoreLogic.pm

```diff
@@ -3449,6 +3450,20 @@ sub processRepairAuto {
 	}
 }
 
+sub processSendIgnoreAll {
+	return if($net->getState() != Network::IN_GAME || !$config{ignoreAll} || $ignored_all);
+
+	if(!$timeout{'ai_ignoreAll'}{'time'}) {
+		$timeout{'ai_ignoreAll'}{'time'} = time;
+		return;
+	} elsif(timeOut($timeout{ai_ignoreAll})) {
+		warning "sending ignoreAll... \n";
+		$messageSender->sendIgnoreAll(0);
+		$timeout{'ai_ignoreAll'}{'time'} = time;
+		$ignored_all = 1;
```

Comment:
i only ask to move this to above the `$messageSender->sendIgnoreAll(0);` line.
like yesterday PR, if someone use this packet as hook, it will get the old value instead of the updated one